types/experiments.ts

```diff
@@ -72,6 +72,13 @@ interface BucketConfig {
   total: number;
 }
 
+interface FeatureConfig {
+  featureId: "cfr" | "aboutwelcome";
+  enabled: boolean;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  value: { [key: string]: any } | null;
```

Comment:
Previously (in this diff) we were using `[key: string]: unknown` and switched it to `unknown` => `any` here. Was there a reason for the change?
Prior to that, it looks like we changed it from `any` => `unknown` in https://github.com/mozilla/nimbus-shared/commit/2bdb8690a7385739aaf0fed4d7a0f26c7515b11c#diff-abab96a820f827ebad3b6a423a30b3f3

Reply:
Matching m-c https://searchfox.org/mozilla-central/rev/0c682c4f01442c3de0fa6cd286e9cadc8276b45f/toolkit/components/messaging-system/experiments/@types/ExperimentManager.d.ts#4